### PR TITLE
Translation README to French / Traduction du README en français

### PR DESCRIPTION
**Pull request by abdelq**
This translation takes into account the edit made to the **Running** section in #1317.

### CODE REVIEW

**Comment by SirCmpwn** [comment]
Merci! I will review this in detail tomorrow.

File: README.fr.md [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+Lisez la [FAQ](https://github.com/SirCmpwn/sway/wiki). Rejoignez le
+[canal IRC](http://webchat.freenode.net/?channels=sway&uio=d4) (#sway sur
+irc.freenode.net).
+
+[![](https://sr.ht/ICd5.png)](https://sr.ht/ICd5.png)
+
+Si vous souhaitez soutenir le développement de Sway, vous pouvez contribuer à [ma page
+Patreon](https://patreon.com/sircmpwn) ou aux [primes](https://github.com/SirCmpwn/sway/issues/986)
+pour des fonctionnalités spécifiques.
+Tout le monde est invité à réclamer une prime et vous pouvez faire un don pour n'importe quel fonctionnalité
```
Comment (4e554c4c) [inline README.fr.md]: s/quel/quelle/

File: README.fr.md [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+
+## Configuration
+
+Si vous utilisez déjà i3, copiez votre configuration i3 à `~/.config/sway/config` et
+ça va fonctionner. Sinon, copiez l'exemple de fichier de configuration à
+`~/.config/sway/config`. Il se trouve généralement dans `/etc/sway/config`.
+Exécutez `man 5 sway` pour l'information sur la configuration.
+
+Mes propres dotfiles sont disponibles [ici](https://git.sr.ht/~sircmpwn/dotfiles) si
+vous voulez un peu d'inspiration, et consultez certainement le
```
Comment (4e554c4c) [inline README.fr.md]: s/consultez certainement/je vous recommande de consulter/

File: README.fr.md [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+    sudo setcap cap_sys_tty_config=eip /usr/local/bin/sway
+
+Sur les systèmes sans logind, vous devez suid le binaire de sway :
+
+    sudo chmod a+s /usr/local/bin/sway
+
+## Configuration
+
+Si vous utilisez déjà i3, copiez votre configuration i3 à `~/.config/sway/config` et
+ça va fonctionner. Sinon, copiez l'exemple de fichier de configuration à
```
Comment (4e554c4c) [inline README.fr.md]: s/ça/cela

File: README.fr.md (hunk 1 above)
Comment (4e554c4c) [inline README.fr.md]: I would say 'donner une prime' instead of 'faire un don' as well

**Comment by abdelq** [comment]
@4e554c4c Thanks!

**Comment by SirCmpwn** [comment]
Please add details on availability of support in French. Can you volunteer to provide support in the IRC channel for French users? Either way, mention it in the README.

**Comment by SirCmpwn** [comment]
In the Japanese readme I said "私の名前はSirCmpwnです。助けのはIRCとGitHubで日本語で頑張ります", which introduces my IRC handle and says I will do my best to help in Japanese on IRC or GitHub. I also said later, regarding the config file: 英語は難しいのばIRCで助けのを尋ねる。￼日本語助けのはUTC-4。Which means "ask in IRC if you have trouble with the english config file, japanese help is available in the UTC-4 timezone".

**Comment by SirCmpwn** [comment]
Can you sync with master and update the links at the top of the english readme as well?

**Comment by SirCmpwn** [comment]
I have updated the i18n guidelines here, please review: https://github.com/SirCmpwn/sway/issues/1318

File: README.fr.md [hunk 4]
```diff
@@ -0,0 +1,97 @@
...
+* pcre
+* json-c
+* pango
+* cairo
+* gdk-pixbuf2 *
+* pam **
+* imagemagick (requis pour la capture d'image avec swaygrab)
+* ffmpeg (requis pour la capture vidéo avec swaygrab)
+
+_\*Uniquement Requis pour swaybar, swaybg, and swaylock_
```
Comment (4e554c4c) [inline README.fr.md]: I don't think that 'Requis' should be capitalized

Comment (4e554c4c) [inline README.fr.md]: Unnecessary capitalization

**Comment by 4e554c4c** [comment]
Thanks so much for the translation!

**Comment by SirCmpwn** [comment]
The guidelines have been updated once again - sorry! These should be sufficient: https://github.com/SirCmpwn/sway/issues/1318.

**Comment by abdelq** [comment]
@SirCmpwn Included the help subsection and added clarification about emailing you in English.

**Comment by SirCmpwn** [comment]
Sorry - please move it to immediately following the donation information.

**Comment by abdelq** [comment]
Done.

**Comment by SirCmpwn** [comment]
Merci!

**Pull request merged**